templates/registration/password_reset_email.txt

```diff
@@ -1,7 +1,7 @@
 Psst. Word on the street is that you forgot your password, {{ email}}.
 
 It's all good. Follow the link below and we'll take care of the rest:
-    <{{ protocol}}://{{ domain }}{{ url('django.contrib.auth.views.password_reset_confirm', kwargs=dict(uidb64=uid, token=token)) }}>
+{{ protocol}}://{{ domain }}{{ url('django.contrib.auth.views.password_reset_confirm', kwargs=dict(uidb64=uid, token=token)) }}
```

Comment:
Cool.  Agree the `<>` aren't needed, but I think a blank line before this (which would be new) is probably best practice for password reset links; what are your thoughts?